pom.xml

```diff
@@ -165,6 +166,34 @@ LICENSE file.
       </plugins>
     </pluginManagement>
     <plugins>
+      <!-- Used to execute junit tests but crashes on some bindings.
```

Comment:
This removed testing?

Reply:
I removed the comment. Strangely JUnit tests in the core module are not executed with `mvn clean package` but in other packages, JUnit tests are run (tested with accumolo1.6).